src/PrivateDns/PrivateDns/help/Remove-AzPrivateDnsVirtualNetworkLink.md

```diff
@@ -0,0 +1,236 @@
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: Object
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PassThru
+passthru
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+Specifies the name of the resource group that contains the link to remove.
+You must also specify the *ZoneName* and *Name* parameter.
+Alternatively, you can specify the DNS zone using a **PSPrivateDnsLink** object, passed via either the pipeline or the *Link* parameter.
+
+```yaml
+Type: System.String
+Parameter Sets: Fields
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceId
+Specifies the ARM resource ID of the link.
+
+```yaml
+Type: System.String
+Parameter Sets: ResourceId
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ZoneName
+Specifies the name of the private DNS zone that the link is associated with.
+You must also specify the *ResourceGroupName* and *Name* parameter.
+Alternatively, you can specify the link using the *Link* parameter.
+
+```yaml
+Type: System.String
+Parameter Sets: Fields
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -WhatIf
+Shows what would happen if the cmdlet runs.
+The cmdlet is not run.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases: wi
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### Microsoft.Azure.Commands.PrivateDns.Models.PSPrivateDnsLink
+
+### System.String
+
+## OUTPUTS
+
+### System.Boolean
+
+## NOTES
+Due to the potentially high impact of deleting a virtual network link, by default, this cmdlet prompts for confirmation if the $ConfirmPreference Windows PowerShell variable has any value other than None.
```

Comment:
Is this true?

Reply:
removed this as we are not using confirmImpact anymore.